notebooks/util.py

```diff
@@ -12,4 +12,12 @@ def is_ncar_host():
     
     return any([re.compile(ncar_host).search(hostname) 
                 for ncar_host in ['cheyenne', 'casper', 'hobart']])
-    
\ No newline at end of file
+
+def create_dask_cluster():
+    if is_ncar_host():
+        from dask_jobqueue import PBSCluster
+        return PBSCluster()
```

Comment:
Since the DAV and Cheyenne batch nodes use incompatible queuing systems, I think it's preferable to do this:
```
    from ncar_jobqueue import NCARCluster
    return NCARCluster(**kwargs)
```
which passed on to `PBSCluster` or `SLURMCluster` as appropriate.